apps/worker/src/app/workflow/usecases/send-message/send-message-sms.usecase.ts

```diff
@@ -137,7 +137,7 @@ export class SendMessageSms extends SendMessageBase {
 
     await this.sendSelectedIntegrationExecution(command.job, integration);
 
-    const overrides = command.overrides[integration?.providerId] || {};
+    const overrides = command.overrides[integration?.channel] || {};
```

Comment:
Why was this changed from `providerId` to `channel`?

Reply:
we dont pass providerId in the overrides, we pass channel
ex:
```
{
 overrides:{sms:{...data}}
}
```
not
```
{
 overrides:{twillio:{...data}}
}
```
i'm not sure why it was written like that, but it was not being used until now, so it its safe to change it.

Reply:
https://github.com/novuhq/novu/blob/e54550062cc5fe2b48f5ba353ff7f2298a69174a/packages/node/src/lib/events/events.interface.ts#L43

https://github.com/novuhq/novu/blob/e54550062cc5fe2b48f5ba353ff7f2298a69174a/apps/worker/src/app/workflow/usecases/send-message/send-message-email.usecase.ts#L147

@mahendraHegde We do support providerId in the overrides for email, it would be great if we can enable the same for sms

Reply:
@BiswaViraj  thanks for pointing that out, updated the PR